app/templates/ratemypdf_stats.html

```diff
@@ -236,6 +277,161 @@ <h2 class="pdf-section-heading">Suggestions</h2>
+                Sentences with passive voice
+              </button>
+            </h2>
+            <div id="collapse-passive-voice" class="accordion-collapse collapse" aria-labelledby="heading-passive-voice" data-bs-parent="#suggestionsAccordion">
+              <div class="accordion-body">
+                <ul>
+                {% for sentence in stats.get("passive sentences", []) %}
+                <li>
+                  {{ highlight_text(sentence[0], sentence[1])  | safe }}
+                </li>
+                {% endfor %}
+                </ul>
+              </div>
+            </div>
+          </div>
+          {% endif %}
+          {% if stats.get("difficult words", []) %}
+          <div class="accordion-item">
+            <h2 class="accordion-header" id="heading-difficult-words">
+              <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-difficult-words" aria-expanded="false" aria-controls="collapse-difficult-words">
+                Consider replacing difficult words
+              </button>
+            </h2>
+            <div id="collapse-difficult-words" class="accordion-collapse collapse" aria-labelledby="heading-difficult-words" data-bs-parent="#suggestionsAccordion">
+              <div class="accordion-body">
+                We refer to [plainlanguage.gov](https://www.plainlanguage.gov/guidelines/words/)'s list of common
+                complex words and simple replacements. Apply judgement and read the words in context to make sure the replacements
+                still make sense.
+                <ul>
+                {% for word in stats.get("difficult words", []) %}
+                <li>
+                  {{ word  }}
+                </li>
+                {% endfor %}
+                </ul>
+              </div>
+            </div>
+          </div>
+          {% endif %}
+          {% if stats.get("plain language suggestions") %}
+          <div class="accordion-item">
+            <h2 class="accordion-header" id="heading-plain-lang-suggestions">
+              <button class="accordion-button collapsed" type="button" data-bs-toggle="collapse" data-bs-target="#collapse-plain-lang-suggestions" aria-expanded="false" aria-controls="collapse-plain-lang-suggestions">
+                Consider substituting simpler words and phrases
+              </button>
+            </h2>
+            <div id="collapse-plain-lang-suggestions" class="accordion-collapse collapse" aria-labelledby="heading-plain-lang-suggestions" data-bs-parent="#suggestionsAccordion">
+              <div class="accordion-body">
+                <table class="table">
+                  <thead>
+                    <tr>
+                      <th scope="col">Original sentence</th>
+                      <th scope="col">Revised sentence</th>
```

Comment:
These columns are off in my version, one gets shoved way off the screen
![Screenshot from 2023-04-21 10-35-48](https://user-images.githubusercontent.com/6252212/233663819-4d88bc3e-224c-4218-814b-e386195727f6.png)

Reply:
I'll check but did you try a force reload? Maybe cached css?

Reply:
Good idea, but it still there.

Reply:
I'm seeing it now too but I'm not sure what's happening. I checked and closed up some mis-closed elements. It looks right when I have the exact same markup but iterate over "neutral gender suggestions" and breaks when I iterate over "plain language suggestions"

Reply:
Turned out to be an issue with long words and things like `________`. Adding `text-break` class fixed it